scripts/common.js

```diff
@@ -6,6 +8,336 @@ export let commonOptions = {
     ]
 };
 
+export function summaryFormatter(data) {
+    // removing default metrics
+    delete data.metrics['http_req_duration']
+    delete data.metrics['http_req_sending']
+    delete data.metrics['http_req_receiving']
+    delete data.metrics['http_req_blocked']
+    delete data.metrics[`http_req_duration{expected_response:true}`]
+    delete data.metrics['http_req_waiting']
+    delete data.metrics['http_reqs']
+    delete data.metrics['iteration_duration']
+    delete data.metrics['iterations']
+    delete data.metrics['vus']
+    delete data.metrics['http_req_connecting']
+    delete data.metrics['http_req_failed']
+    delete data.metrics['http_req_tls_handshaking']
+
+    data = destinations.reduce((curData, curDestinations) => {
+        return reportPerDestination(curData, curDestinations)
+    }, data)
+
+    return {
+        stdout: textSummary(data, { indent: ' ', enableColors: true }),
+    }
+}
+
+function reportPerDestination(data, destinations) {
+    data.metrics[`http_req_sending(${destinations} destinations)`] = data.metrics[`http_req_sending{scenario:sending_${destinations}_destinations}`]
+    delete data.metrics[`http_req_sending{scenario:sending_${destinations}_destinations}`]
+    data.metrics[`http_req_receiving(${destinations} destinations)`] = data.metrics[`http_req_receiving{scenario:sending_${destinations}_destinations}`]
+    delete data.metrics[`http_req_receiving{scenario:sending_${destinations}_destinations}`]
+    data.metrics[`http_req_duration(${destinations} destinations)`] = data.metrics[`http_req_duration{scenario:sending_${destinations}_destinations}`]
+    delete data.metrics[`http_req_duration{scenario:sending_${destinations}_destinations}`]
+    return data
+}
+
+export const destinations = (__ENV.DESTINATIONS || '50, 100, 150')
+    .split(',')
+    .map((curDestinations) => parseInt(curDestinations))
+
+const scenarios = destinations.reduce((accumulator, currentDestinations) => {
```

Comment:
if you move it to a common file, then remove  it from time-filter-proto.

Reply:
Done